docs/private-networks/get-started/start-node.md

```diff
@@ -104,3 +104,10 @@ Where `<data-path>` is the path to the directory to save the chain data to. Ensu
 You might need to set [`--tx-pool-limit-by-account-percentage`](../../public-networks/reference/cli/options.md#tx-pool-limit-by-account-percentage) to 1. The default value is suitable for Mainnet, but may cause issues on private networks.
 
 :::
+
+:::note Sync nodes for BFT
+
+If you're using a Byzantine Fault Tolerance (BFT) consensus mechanism such as QBFT or IBFT 2.0, set your node's [synchronization mode](/public-networks/get-started/connect/sync-node#run-a-full-node) to either `FAST` or `FULL`. 
+By default, the configuration is set to `--sync-mode=FAST`.
```

Comment:
```suggestion
If you're running a node on a [QBFT](../how-to/configure/consensus/qbft.md) or [IBFT 2.0](../how-to/configure/consensus/ibft.md) network, your node must use [fast sync](/public-networks/get-started/connect/sync-node#fast synchronization) or [full sync](/public-networks/get-started/connect/sync-node#run-an-archive-node). 
Full sync is set by default.
```

Simplify and be specific. QBFT and IBFT 2.0 are the specific and only options this applies to. Original wording implies you have to set your sync mode – make more clear that you don't need to change anything if you want to use full sync. Also this is confusing but "full sync" = "archive node" not "full node", so make sure you're linking to right section.

Reply:
Maybe I am not understanding this correctly, but I thought QBFT and IBFT were not networks but a way/ method to secure a network?

Reply:
Thanks @alexandratran for correcting/clarifying. A few things :

- I don't think we mention what BFT is anywhere in the Besu docs (we have procs on QBFT, but don't explicitly spell out the acronym). I know for an experienced developer they will know what it is , but from the customer journey perspective of a newer user, might it be helpful to explain that these are "Byzantine Fault Tolerance (BFT) consensus mechanisms" (or maybe include this in the conceptual page for QBFT and IBFT). Also, maybe for SEO, it might be helpful to keep this in. 

- Also, is it correct that QBFT and IBFT are not networks, but instead ways to secure within a network? I know in the docs we have it as QBFT networks, but I found this a bit confusing. Especially when onboarding.  I think it might be helpful (also from a localization standpoint) to maybe make this clearer. 

I know this is longer (just tweaked your suggestions slightly), but is this okay?

```suggestion
If you're running a node with the Byzantine Fault Tolerance (BFT) consensus mechanism  [QBFT](../how-to/configure/consensus/qbft.md) or [IBFT 2.0](/how-to/configure/consensus/ibft.md), you must set your node's synchronization mode to either [`FAST`](/public-networks/get-started/connect/sync-node#fast-synchronization) or [`FULL`](/public-networks/get-started/connect/sync-node#run-an-archive-node). 

Full sync is set by default.
```

or maybe I could move the "Byzantine Fault Tolerance (BFT) consensus mechanism" to the actual QBFT and IBFT pages?

Reply:
:::note Sync nodes for BFT

If you're running a node on a [QBFT](../how-to/configure/consensus/qbft.md) or [IBFT 2.0](../how-to/configure/consensus/ibft.md) network, your node must use [fast sync](../../public-networks/get-started/connect/sync-node#fast-synchronization) or [full sync](../../public-networks/get-started/connect/sync-node#run-an-archive-node). 

Full sync is set by default.

:::